test/specs/sause-login/placeholder-text-and-credentials.spec.js

```diff
@@ -0,0 +1,55 @@
+import { assert } from 'chai'
+
+describe('Test case 2', function () {
+  it('Should verify placeholder text and credentials', async function () {
+    // Open https://www.saucedemo.com/.
+    const url = 'https://www.saucedemo.com/'
+    await browser.url(url)
+
+    // Confirm the login page is opened.
+    assert.equal(await browser.getUrl(), url, 'URL is not as expected.')
+
+    // Verify the placeholder text for the username field is "Username".
+    const username = await $('//input[@data-test="username"]')
+    const usernamePlaceholder = await username.getAttribute('placeholder')
+    assert.equal(usernamePlaceholder, 'Username', 'Placeholder should be "Username".')
+
+    // Confirm the placeholder text for the password field is "Password".
+    const password = await $('//input[@data-test="password"]')
+    const passwordPlaceholderText = await password.getAttribute('placeholder')
+    assert.equal(passwordPlaceholderText, 'Password', 'Placeholder should be "Password".')
+
+    // Check the text on the "Login" button is "Login".
+    const loginButton = await $('//input[@data-test="login-button"]')
+    const loginButtonText = await loginButton.getValue()
+    assert.equal(loginButtonText, 'Login', 'Button text should be "Login".')
+
+    // Verify the color of the "Login" button is green (#3ddc91).
+    const color = await loginButton.getCSSProperty('background-color')
+    const actualColor = color.parsed.hex
+    const expectedColor = '#3ddc91'
```

Comment:
Usually on a project if you have such kind of validations you have the list on constants with colors. Please extract this to a some object  as you did with timeouts

Reply:
Added a file with color contents corresponding to page. Can you please merge my branch?

Reply:
Please rename them to smth like GREEN, RED, etc. Do you think it makes much sense to use constant like BUTTON_BACKGROUND for checking some text color on the page let's say.